content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -87,18 +87,24 @@ The inventory valuation report
 ------------------------------
 
 To view the current value of all products in the warehouse, go to :menuselection:`Inventory app -->
-Reporting --> Inventory Valuation`. The records in the table are organized by product, and selecting
-a product's drop-down menu displays detailed records with the :guilabel:`Date`,
-:guilabel:`Quantity`, and :guilabel:`Total Value` of the inventory.
+Reporting --> Valuation`. The page displays detailed records of products with the :guilabel:`Date`,
+:guilabel:`Quantity`, :guilabel:`Unit Value` and :guilabel:`Total Value` of the inventory.
+
+.. important::
+   :ref:`Developer mode<developer-mode>` must be turned on to see the :guilabel:`Valuation` option
+   under :guilabel:`Reporting`.
 
 .. image:: using_inventory_valuation/inventory-valuation-products.png
    :align: center
    :alt: Inventory valuation report showing multiple products.
 
-.. note::
-   The :guilabel:`Inventory At Date` button, located in the top-left corner of the :guilabel:`Stock
-   Valuation` dashboard, shows the inventory valuation of products available during a prior
-   specified date.
+The :guilabel:`Inventory At Date` button, located in the top-left corner of the :guilabel:`Stock
+Valuation` dashboard, shows the inventory valuation of products available during a prior specified
+date.
```

Comment:
```suggestion
The :guilabel:`Valuation At Date` button, located in the top-left corner of the :guilabel:`Stock
Valuation` page, reveals a pop-up window. In this pop-up, the inventory valuation of products
available during a prior specified date can be selected and seen.
```
the name of the button in my v16 runbot was "Valuation At Date" (not "Inventory At Date") - added the mention of the pop-up window that appears, as well